README.md

```diff
@@ -1,2 +1,20 @@
 # ARGOS
+
 Auditing Reporter for Greener Ops Strategies
+
+## Requirements
+
+- Docker
+- Docker compose
+
+## Installation
+
+`make install`
+
+## Usage
+
+`NAME="[name_of_the_run]" docker-compose up`
```

Comment:
Would be great if it was a make command.

Reply:
You're right. It might be done after the server refactor @ThieryMichel  is working on because if I do it now, it will be erased.